tests/test_fields.py

```diff
@@ -13,6 +14,7 @@
 
 import rest_framework
 from rest_framework import exceptions, serializers
+from rest_framework.exceptions import ValidationError
```

Comment:
```suggestion
```
Looks like you're using the one on the `serializer` module so you don't need to re-import it.